src/main/java/com/squareup/kotlinpoet/FunSpec.kt

```diff
@@ -283,6 +297,28 @@ class FunSpec private constructor(builder: Builder) {
       parameters += parameterSpec
     }
 
+    fun callThisConstructor(vararg args: String) = apply {
+      callConstructor("this", *args.map { CodeBlock.of(it) }.toTypedArray())
+    }
+
+    private fun callConstructor(constructor: String, vararg args: CodeBlock) {
```

Comment:
Can you move this to the bottom of all four methods. Overloads have to stay next to each other and I'd prefer the this/super functions also be grouped.

Reply:
Yep, done